### DISCUSSION TITLE
Add related tab multiple times

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#14298] [How to]
Having need to add related tab multiple time, but module-relations overwriting already existing relation. I can do in database, but wondering why not allow it via  module-relations settings?

**Comment by mariuszkrzaczkowski** [comment]
because it requires reworking the panel, we tested many relationships to the same module and it works

**Reply by TonyM15** [reply]
I have testes this too and tabs related to the same module with the same relation type (many to many or with separation data) returning the same set of records. So if we want organize related records of the same module in tabs - let say by some status - it will not work. Someone could say to use filters, but this is very different scenario
